extensions/sdk/src/Rally.ts

```diff
@@ -119,14 +119,13 @@ export class Rally {
     const camelToHyphenCase = str => str.replace(/[A-Z]/g, letter => `-${letter.toLowerCase()}`);
     const storeId = camelToHyphenCase(this._options.studyId);
 
-    let storeUrl;
-    if ("getBrowserInfo" in browser.runtime) {
-      const browserInfo = await browser.runtime.getBrowserInfo();
-      if (browserInfo.name === "firefox") {
-        storeUrl = `https://addons.mozilla.org/en-US/firefox/addon/${storeId}/*`
-      }
-    } else {
-      storeUrl = `https://chrome.google.com/webstore/detail/${storeId}/*`;
+
+    let storeUrl = `https://chrome.google.com/webstore/detail/${storeId}/*`; // Default
```

Comment:
Minor: Literals can be extracted as consts at the top of the file:
const ChromeStoreUrl = ...